internal/config/config.go

```diff
@@ -188,12 +206,66 @@ func Read() (HCCMConfiguration, error) {
 		errs = append(errs, err)
 	}
 
+	cfg.LoadBalancer.ProxyProtocolEnabled, err = getEnvBoolPtr(hcloudLoadBalancersUsesProxyProtocol)
+	if err != nil {
+		errs = append(errs, err)
+	}
+
 	disableIPv6, err := getEnvBool(hcloudLoadBalancersDisableIPv6, false)
 	if err != nil {
 		errs = append(errs, err)
 	}
 	cfg.LoadBalancer.IPv6Enabled = !disableIPv6 // Invert the logic, as the env var is prefixed with DISABLE_.
 
+	if subnetRange, ok := os.LookupEnv(hcloudLoadBalancersPrivateSubnetIPRange); ok {
+		cfg.LoadBalancer.PrivateSubnetIPRange = subnetRange
+	}
+
+	if algorithmType, ok := os.LookupEnv(hcloudLoadBalancersAlgorithmType); ok {
+		alg, parseErr := parseLoadBalancerAlgorithmType(algorithmType)
+		if parseErr != nil {
+			errs = append(errs, fmt.Errorf("failed to parse %s: %w", hcloudLoadBalancersAlgorithmType, parseErr))
+		} else {
+			cfg.LoadBalancer.AlgorithmType = alg
+		}
+	}
+
+	if interval, ok := os.LookupEnv(hcloudLoadBalancersHealthCheckInterval); ok {
+		d, parseErr := time.ParseDuration(interval)
+		if parseErr != nil {
+			errs = append(errs, fmt.Errorf("failed to parse %s: %w", hcloudLoadBalancersHealthCheckInterval, parseErr))
+		} else {
+			cfg.LoadBalancer.HealthCheckInterval = d
+		}
+	}
```

Comment:
```suggestion
    cfg.LoadBalancer.HealthCheckInterval, err = getEnvDuration(hcloudLoadBalancersHealthCheckInterval)
    if err != nil {
		errs = append(errs, err)
    }
```

We already have a util function for this. If `errs` is not empty, HCCM won't continue and throw an error. So the `if-else` is not necessary here.